features/admin_clicks_cat_to_see_details.feature

```diff
@@ -0,0 +1,89 @@
+Feature: Link categories to categories show
```

Comment:
add " for an Admin" to the feature description  (name) so it's clear right in that description that this is limited to the admin.

I know it states it in the "As an...." below, but I think it should be in the Feature name.
Others might disagree.